### DISCUSSION TITLE
obstacles in Roborock App

### DISCUSSION THREAD

**Discussion by poudenes** [humbertogontijo/homeassistant-roborock#515] [Q&A]
Hi all,

I removed lots of old obstacles in the Roborock App. But those are still visible on the MAP in HA?
How can i remove them also on the MAP in HA?

**Comment by humbertogontijo** [comment]
Map only refreshes when the robot is cleaning

**Comment by poudenes** [comment]
Then there is a bug. Because my robot cleans every day and the map is not refreshing.

**Reply by humbertogontijo** [reply]
Your map is not refreshing when the robot is cleaning?

**Reply by poudenes** [reply]
The map is refreshing but the obstacles ar not removed from the map:

Roborock App:
![Schermafbeelding 2023-10-17 om 16 17 05](https://github.com/humbertogontijo/homeassistant-roborock/assets/4399532/718e1a97-a4c5-4af7-9d1b-b1ba8386d9d8)

Map in HA:
<img width="485" alt="Screenshot 2023-10-17 at 16 15 53" src="https://github.com/humbertogontijo/homeassistant-roborock/assets/4399532/09179873-883d-4412-a91c-f62d632ce6aa">

**Reply by Derezzler** [reply]
Did you ever figure out how to fix this?

**Reply by poudenes** [reply]
> Did you ever figure out how to fix this?

Nope... Obstacles still there and in meanwhile the roborock cleaned my house many times
